### PR TITLE
Setup Rubocop

### PR DESCRIPTION
**Pull request by bforma**

### CODE REVIEW

File: spec/bunq/payments_spec.rb [hunk 1]
```diff
@@ -36,12 +38,13 @@
...
-            "Payment": { "id": 42 },
+            "Payment": {"id": 42},
           },
           {
-            "Payment": { "id": 84 },
+            "Payment": {"id": 84},
           },
-        ])
+        ],
+                                            )
```
Comment (lvonk) [inline spec/bunq/payments_spec.rb:47]: Weird indentation.

File: spec/bunq/monetary_account_spec.rb [hunk 2]
```diff
@@ -10,11 +12,12 @@
...
-      stub_request(:get, "#{user_url}/monetary-account/#{account_id}").
-        to_return({
+      stub_request(:get, "#{user_url}/monetary-account/#{account_id}")
+        .to_return({
           status: status_code,
-          body: response
-        })
+          body: response,
+        },
+                  )
```
Comment (lvonk) [inline spec/bunq/monetary_account_spec.rb:20]: Weird indentation.

File: spec/bunq/payments_spec.rb (hunk 1 above)
Comment (bforma) [reply spec/bunq/payments_spec.rb:47]: Yes, noticed that too, will fix it.

File: spec/bunq/monetary_account_spec.rb (hunk 2 above)
Comment (bforma) [reply spec/bunq/monetary_account_spec.rb:20]: Yes, noticed that too, will fix it.

**Comment by lvonk** [comment]
And update changelog ;-)

**Pull request merged**